DOCUMENTATION.md

```diff
@@ -2432,6 +2432,12 @@ dashboardDeviceUrl = balena.models.device.getDashboardUrl('a44b544b8cc24d11b036c
 <a name="balena.models.device.getAll"></a>
 
 ##### device.getAll([options]) ⇒ <code>Promise</code>
+This method returns all devices that the current user can access.
+In order to have the following computed properties filled in the result
+you have to explicitly define then in a `$select` in the extra options:
```

Comment:
```suggestion
you have to explicitly define them in a `$select` in the extra options:
```

Reply:
I also had backticks, instead of single quotes in the `$select`. I guess my eyes were getting tired.

Reply:
I'll just pre-approve in case I am not around.